src/adapters/peggedAssets/euroe-stablecoin/index.ts

```diff
@@ -15,6 +15,9 @@ type ChainContracts = {
 const chainContracts: ChainContracts = {
   ethereum: {
     issued: ["0x820802Fa8a99901F52e39acD21177b0BE6EE2974"],
+  },
+  polygon: {
+    issued: ["0x820802Fa8a99901F52e39acD21177b0BE6EE2974"],
```

Comment:
polygon: {
       bridgedFromETH: "0x820802Fa8a99901F52e39acD21177b0BE6EE2974",
    },

Reply:
Hey @Define101 – the EUROe on Polygon is not bridged! It is natively issued on Polygon. For more information, please see https://dev.euroe.com/docs/Stablecoin/contract-addresses#polygon-addresses